### PR TITLE
Adding square-root exercise

### LINKED ISSUE #906: Add new exercise: square-root
[problem specification ](https://github.com/exercism/problem-specifications/tree/master/exercises/square-root)| [description](https://github.com/exercism/problem-specifications/blob/master/exercises/square-root/description.md) | [canonical data](https://github.com/exercism/problem-specifications/blob/master/exercises/square-root/canonical-data.json)

### PR DESCRIPTION
**Pull request by evelynstender**
Closes #906

### CODE REVIEW

File: exercises/square-root/example.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export const squareRoot = (radicand = 0) => {
+  return Math.sqrt(radicand);
```
Comment (SleeplessByte) [inline exercises/square-root/example.js:2]: I think the point is to _not_ use a built-in function 😅

**Review by SleeplessByte — COMMENTED** [review]
I think we should clarify we don't want to use `Math.sqrt`. We might check other languages that have implemented this exercise to confirm that other tracks also disallow using built-in functions.

If this is the case:

- add `.meta/hints.md` and disallow `Math.sqrt`
- `bin/configlet generate . --only square-root` again to update the README.md
- update the example solution

**Comment by evelynstender** [comment]
Oh yeah! That makes sense! I didn't realise we couldn't use `Math.sqrt()`. I didn't see it anywhere haha

I will update! Thanks!

File: config.json [hunk 2]
```diff
@@ -227,6 +227,14 @@
       "difficulty": 1,
       "topics": ["booleans", "integers", "logic"]
     },
+    {
+      "slug": "square-root",
+      "uuid": "31c7725e-def0-4dc9-88a0-e19afb9bfb06",
+      "core": false,
+      "unlocked_by": null,
```
Comment (evelynstender) [inline config.json:234]: I'm not sure which exercise should unlock this (if any)

**Comment by evelynstender** [comment]
@SleeplessByte I updated the example, added the `hint.md` and updated the readme file. I'm not sure which exercise should unlock this (if any), so I left it as `null`

**Review by SleeplessByte — APPROVED** [review]
Perfect!

**Comment by SleeplessByte** [comment]
@evelynstender in v2, _only_ core exercises can unlock anything, and since this is a reimplementation and mathy exercise, it will never become core.

**Pull request merged**